### PR TITLE
Stats: Use enum types instead of match pattern.

### PR DESCRIPTION
**Pull request by songy23**
Fixes https://github.com/bogdandrutu/openconsensus/issues/48.

/cc @bogdandrutu @SergeyKanzhelev

### CODE REVIEW

File: api/src/main/java/openconsensus/stats/data/MeasureType.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+package openconsensus.stats.data;
+
+/**
+ * An enum that represents all the possible value types for a {@code Measure} or a {@code
+ * Measurement}.
+ *
+ * @since 0.1.0
+ */
+public enum MeasureType {
```
Comment (bogdandrutu) [inline api/src/main/java/openconsensus/stats/data/MeasureType.java]: Move type in the measure.

Comment (songy23) [reply api/src/main/java/openconsensus/stats/data/MeasureType.java]: Done. Note this `Type` is also used in Measurement.

**Comment by songy23** [comment]
Please help me merge this PR as I don't have write access to this repo.

**Comment by songy23** [comment]
Never mind - now I have it :)

**Pull request merged**